client/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "academic-team-management-ui",
   "version": "0.1.0",
   "private": true,
+  "proxy": "http://localhost:5001",
```

Comment:
that proxy already exists no? it's under setupProxy.js in client

Reply:
yeah right I cleaned it up already

Reply:
🚀